src/data/event.js

```diff
@@ -77,3 +77,11 @@ export type Asset = {
 };
 
 export type EventDays = Event[][];
+
+export type EventCategory = {
+  label: string,
+  color: string,
+  contrast?: boolean
```

Comment:
I suggest making this required instead of optional and relying on undefined, then just add false to the categories that should not invert the color. Reason being this can now be three different values, `true` `false` and `undefined`. Makes it much clearer that it can only be two values `true` and `false`.